draft-gpew-priv-ppm.md

```diff
@@ -692,8 +704,9 @@ continuation of a previous sub-batch?]
 
 ### Aggregate Request
 
-The AggregateReq request is used by the leader to send a set of reports to the
-helper. These reports MUST all be associated with the same PPM task and batch.
+The AggregateReq request is used by the leader to coordinate aggregation of a
+set of reports with the helpers. These reports incident to a particular
+AggregateReq MUST all be associated with the same PPM task and batch.
```

Comment:
Now that the helper is responsible for its own nonce, report and aggregate accumulator storage, do we need aggregate requests to not span batch intervals anymore? I'm not sure but I think the reason we had this constraint on aggregate requests was so that the state blob could be "sharded" by batch interval and allow parallelism in the helper. Now, though, we might expect the helper to have its own storage where it keeps track of aggregate accumulators for several different batch intervals.

Reply:
Yeah, it may not be strictly necessary. However I think we should keep this as-is for now in order to keep this PR minimal. Plus, it may be helpful for something we're not seeing right now.

Reply:
Agreed. Let's note this and revisit in a followup change. (Good catch, Tim!)